From f7eb207a3b90fef6a9c8fbdc506e809cc9757bd1 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Thu, 20 Sep 2018 21:20:16 +0200
Subject: [PATCH] v4l2-ctl: fix glibc 2.28 build

Include header needed for the major/minor macros in glibc 2.28+.

This patch is extracted from upstream commit 65e7b2a4076845d (v4l2-ctl: show
media/entity info if available).

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 utils/v4l2-ctl/v4l2-ctl.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/v4l2-ctl/v4l2-ctl.cpp b/utils/v4l2-ctl/v4l2-ctl.cpp
index e02dc756..12a19bc7 100644
--- a/utils/v4l2-ctl/v4l2-ctl.cpp
+++ b/utils/v4l2-ctl/v4l2-ctl.cpp
@@ -33,6 +33,7 @@
 #include <errno.h>
 #include <sys/ioctl.h>
 #include <sys/time.h>
+#include <sys/sysmacros.h>
 #include <dirent.h>
 #include <math.h>
 
-- 
2.14.4

